src/components/Checkbox/Checkbox.scss

```diff
@@ -1,122 +1,143 @@
 .Checkbox {
     &__box {
-        background-color: var(--color-background);
-        border: 1px solid var(--color-neutral-30);
-        border-radius: var(--border-radius);
-        box-sizing: content-box;
+        background-color: var(--color-background-selected-subtlest-default, #E6F2FF);
+        border: var(--border-width-thick) solid var(--color-border-input, #808080);
+        border-radius: var(--border-width-thick);
+        box-sizing: border-box;
         display: inline-block;
         position: relative;
         top: -1px;
-        width: 12px;
-        height: 12px;
-        transform: scale(1);
+        width: var(--font-size-normal);
+        height: var(--font-size-normal);
         transition: all var(--transition-duration) ease;
         vertical-align: middle;
         margin-right: var(--space-75);
 
-        &--asFlexbox {
-            flex-shrink: 0;
-        }
-
-        &:focus,
-        &:hover {
-            border: 1px solid var(--color-brand-50);
-            stroke: var(--color-highlight-background);
-            background-color: var(--color-highlight-background);
+        & svg {
+            display: none;
         }
 
-        &:before {
-            content: '';
-            width: 100%;
-            height: 100%;
-            background: var(--color-brand-50);
-            display: block;
-            transform: scale(0);
-            opacity: 1;
-            border-radius: 50%;
+        &--asFlexbox {
+            flex-shrink: 0;
         }
     }
 
     &__label {
-        margin: auto;
-        user-select: none;
         cursor: pointer;
-        display: block;
 
         &--asFlexbox {
             display: flex;
             align-items: center;
         }
+
+        &:hover {
+            .Checkbox__box {
+                background: var(--color-background-selected-subtlest-hover, #CCE4FF);
+            }
+        }
+
+        &:active {
+            .Checkbox__box {
+                background: var(--color-background-selected-subtlest-pressed, #99CAFF);
+            }
+        }
     }
 
     &__svg {
         position: absolute;
-        top: 0;
+        top: var(--border-width-thick);
         left: 0;
         fill: none;
-        stroke: var(--color-background);
+        fill-rule:evenodd;
+        clip-rule:evenodd;
+        stroke: var(--color-background-information-subtlest-default);
```

Comment:
shouldn't it be white as mentioned in Dima's comment? 👀

Reply:
When I talked to Dima he said it actually does look ok but I forgot to check after that. Nice, thanks.